src/colors.ts

```diff
@@ -1,14 +1,16 @@
-// TODO: We need better and more colors. Also might become obsolete if we decide to make colors configurable
-const colors: string[] = ['#8884d8', '#1ab015', '#dce320', '#e39820'];
-
-export function generateRandomColor() {
-    // eslint-disable-next-line unicorn/numeric-separators-style
-    return `#${(0x1000000 + Math.random() * 0xffffff).toString(16).slice(1, 7)}`;
-}
+// https://getbootstrap.com/docs/5.2/customize/css-variables/
+// TODO: Might become obsolete if we decide to make colors configurable
+// TODO: Add 3-4 more colors
+const colors: string[] = [
+    'var(--bs-primary)',
+    'var(--bs-yellow)',
+    'var(--bs-gray)',
```

Comment:
I think we should remove gray from here for the following reasons: 
1. In the chart view, deselecting a legend element grays it out, which will become less obvious if it is gray in the first place. 
2. And in the map view, gray is probably not the most visible accent color, since most of the map background is gray aswell.

Reply:
Agreed